src/lib/Scenes/Collection/Components/__tests__/FeaturedArtists-tests.tsx

```diff
@@ -0,0 +1,93 @@
+import { Theme } from "@artsy/palette"
+import { FeaturedArtists_collection } from "__generated__/FeaturedArtists_collection.graphql"
+import { renderRelayTree } from "lib/tests/renderRelayTree"
+import React from "react"
+import { graphql } from "react-relay"
+import { CollectionFeaturedArtistsContainer as FeaturedArtists } from "../FeaturedArtists"
+
+jest.unmock("react-relay")
+
+const CollectionFixture: FeaturedArtists_collection = {
+  artworksConnection: {
+    merchandisableArtists: [
+      {
+        slug: "pablo-picasso",
+        internalID: "1234",
+        name: "Pablo Picasso",
+        image: {
+          resized: {
+            url: "/some/resized/picasso/image/url",
+          },
+        },
+        birthday: "1877",
+        nationality: "American",
+        isFollowed: true,
+      },
+      {
+        slug: "andy-warhol",
+        internalID: "2342",
+        name: "Andy Warhol",
+        image: {
+          resized: {
+            url: "/some/resized/warhol/image/url",
+          },
+        },
+        birthday: "1947",
+        nationality: "American",
+        isFollowed: false,
+      },
+      {
+        slug: "joan-miro",
+        internalID: "3454",
+        name: "Joan Miro",
+        image: {
+          resized: {
+            url: "/some/resized/miro/image/url",
+          },
+        },
+        birthday: "1877",
+        nationality: "Spanish",
+        isFollowed: true,
+      },
+    ],
+  },
+  " $refType": null,
+}
+
+describe("FeaturedArtists", () => {
+  const render = () =>
+    renderRelayTree({
+      Component: ({ marketingCollection }) => (
+        <Theme>
+          <FeaturedArtists collection={marketingCollection} />
+        </Theme>
+      ),
+      query: graphql`
+        query FeaturedArtistsTestsQuery @raw_response_type {
+          marketingCollection(slug: "emerging-photographers") {
+            ...FeaturedArtists_collection
+          }
+        }
+      `,
+      mockData: {
+        marketingCollection: CollectionFixture,
+      },
+    })
+
+  it("renders properly", async () => {
+    const tree = await render()
+    expect(tree.html()).toMatchSnapshot()
+  })
+
+  it("renders an EntityHeader for each featured artist", async () => {
+    const tree = await render()
+
+    const entityHeaders = tree.find("EntityHeader")
+    expect(entityHeaders.length).toEqual(3)
+
+    const output = tree.html()
+    expect(output).toContain("Pablo Picasso")
+    expect(output).toContain("Andy Warhol")
+    expect(output).toContain("Joan Miro")
+  })
+})
```

Comment:
Test coverage looks good 👍

Reply:
> (Though I'd also be okay with merging & then fixing those two things later, if that's your preference @ashleyjelks.)


Ok, let me address the img resize based on the screen width in a follow-up. The other requested changes have been made!